App/Screens/Home/Footer/ShareButton/ShareButton.tsx

```diff
@@ -82,3 +86,13 @@ export function ShareButton(props: ShareButtonProps): React.ReactElement {
 		</View>
 	);
 }
+
+/**
+ * I18NEXT-PARSER
+ * https://github.com/i18next/i18next-parser#caveats
+ *
+ * > const message =
+ * t('sharing.located', 'in {{place}}', {context: 'city'})
+ * t('sharing.located', 'here', {context: 'here'})
+ * t('sharing.message', 'Shoot! People {{city}} \"smoke\" {{amount}} cigarettes daily just by breathing urban air. Try it out yourself: {{link}}')
```

Comment:
what is this commented code for?

Reply:
I'm sorry for such a mess. I was overjoy from successful replace new tool. I must left my guard down.

Ahout this part, I need to open pc to review it. I'll tell you tomorrow.

Reply:
This comment must be kept in order for i18next-parser extract the translation script correctly. The `> const message =` is actually an indication which tell use where the next 4 lines belong. In this case, it belongs to line 63.